isis/src/base/objs/Cube/Cube.h

```diff
@@ -221,9 +221,10 @@ namespace Isis {
          * |     *     *     *     *    |*
          * |****************************|*
          * |Tile *Tile *Tile *Tile *Tile|*
+         * | 10  * 11  * 12  * 13  * 14 |*
+         * |     *     *     *     *    |
          * ------------------------------*
-         * * 10  * 11  * 12  * 13  * 14  *
-         * *******************************
+
```

Comment:
This was just a typo in the comments that was really bugging me. I'd seen it a couple times and decided it was time to fix it